make scheduler throws error on TYPO3 7

When trying to run `make scheduler` the execution ends with an error:

```
make scheduler
rpc error: code = 2 desc = "oci runtime error: exec failed: exec: \"typo3/cli_dispatch.phpsh\": stat typo3/cli_dispatch.phpsh: no such file or directory"Makefile:82: recipe for target 'scheduler' failed
make: *** [scheduler] Error 126
```

The solution seems to modify the command in `Makefile` and remove `-it` from `docker exec`:

```
diff --git a/Makefile b/Makefile
index 928e961..680f636 100644
--- a/Makefile
+++ b/Makefile
@@ -79,7 +79,7 @@ root:
 #############################

 scheduler:
-       docker exec -it $$(docker-compose ps -q app) typo3/cli_dispatch.phpsh scheduler $(ARGS)
+       docker exec $$(docker-compose ps -q app) web/typo3/cli_dispatch.phpsh scheduler $(ARGS)

 #############################
 # Argument fix workaround
```

--- a/Makefile
+++ b/Makefile
@@ -79,7 +79,8 @@ root:
 #############################
 
 scheduler:
-	docker exec -it $$(docker-compose ps -q app) typo3/cli_dispatch.phpsh scheduler $(ARGS)
+	# TODO: remove the workaround "; (exit $?)" when https://github.com/docker/compose/issues/3379 has been fixed
+	docker-compose exec --user application app /bin/bash -c '"$$WEB_DOCUMENT_ROOT"typo3/cli_dispatch.phpsh scheduler $(ARGS); (exit $$?)'
 
 #############################
 # Argument fix workaround

